### PR TITLE
Mobile Inverse Kinematics Problem

### PR DESCRIPTION
**Pull request by cohnt**
This change is [Reviewable](https://reviewable.io/reviews/RussTedrake/manipulation/262)

### CODE REVIEW

File: manipulation/models/marks-table/table.mtl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Blender MTL File: 'edit_shelf_geom.blend'
```
Comment (RussTedrake) [inline manipulation/models/marks-table/table.mtl:1]: do the table models in drake really not work?  i really dislike adding all these asset files; they'll be in git history forever.

File: manipulation/models/pr2_collision_convex/base_v0/base_L.obj [hunk 2]
```diff
@@ -0,0 +1,108 @@
+####
+#
+# OBJ File Generated by Meshlab
```
Comment (RussTedrake) [inline manipulation/models/pr2_collision_convex/base_v0/base_L.obj:3]: why do we need the convex meshes?  if you just mark the file convex, doesn't drake do that for you?

File: manipulation/exercises/mobile/test_mobile_base_ik.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+    @weight(5)
+    @timeout_decorator.timeout(10.0)
+    def test_mobile_base_ik(self):
+        """Test solve_ik"""
+        # yapf: disable
+        goal_rotation = RotationMatrix([  # noqa
+            [0.9969323810421103, -0.0015776037860101618, -0.07825176544975043],
+            [-0.010615080452914535, -0.9932842569807606, -0.11521156583067553],
+            [-0.07754448849974165, 0.11568878943299686, -0.9902539857545847],
```
Comment (RussTedrake) [inline manipulation/exercises/mobile/test_mobile_base_ik.py:21]: i'm curious why you have these crazy looking numbers..? (here and in the solns?)

File: manipulation/models/marks-table/table.mtl (hunk 1 above)
Comment (cohnt) [reply manipulation/models/marks-table/table.mtl]: Switched to using only models from Drake, except for the fixed PR2 urdf.

File: manipulation/models/pr2_collision_convex/base_v0/base_L.obj (hunk 2 above)
Comment (cohnt) [reply manipulation/models/pr2_collision_convex/base_v0/base_L.obj]: Switched back to just using the visual geometry as collision geometry, as in the original drake model.

File: manipulation/exercises/mobile/test_mobile_base_ik.py (hunk 3 above)
Comment (cohnt) [reply manipulation/exercises/mobile/test_mobile_base_ik.py]: I had originally obtained these by teleoperating the robot to some interesting poses and storing them. I've switched to more intuitive poses.

**Review by RussTedrake — CHANGES_REQUESTED** [review]
> Reviewed 1 of 27 files at r1, 26 of 26 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/RussTedrake/manipulation/262)* status: all files reviewed, 3 unresolved discussions (waiting on @cohnt)

___
*[`manipulation/exercises/mobile/test_mobile_base_ik.py` line 21 at r1](https://reviewable.io/reviews/RussTedrake/manipulation/262#gh-1372466577:-Nhg0Tk28Shs_4Jxustl:b-83sjo4) ([raw file](https://github.com/RussTedrake/manipulation/blob/a0db71001c33901a43ec10ddabd322750f4fdae0/manipulation/exercises/mobile/test_mobile_base_ik.py#L21)):*
<details><summary><i>Previously, cohnt (Thomas Cohn) wrote…</i></summary><blockquote>

I had originally obtained these by teleoperating the robot to some interesting poses and storing them. I've switched to more intuitive poses.
</blockquote></details>

Nice.
___
*[`manipulation/models/marks-table/table.mtl` line 1 at r1](https://reviewable.io/reviews/RussTedrake/manipulation/262#gh-1372465088:-Nhg18a1E2APvJiUlKsv:bp0kl5m) ([raw file](https://github.com/RussTedrake/manipulation/blob/a0db71001c33901a43ec10ddabd322750f4fdae0/manipulation/models/marks-table/table.mtl#L1)):*
<details><summary><i>Previously, cohnt (Thomas Cohn) wrote…</i></summary><blockquote>

Switched to using only models from Drake, except for the fixed PR2 urdf.
</blockquote></details>

hoorah!  thanks.
___
*[`manipulation/models/pr2_collision_convex/base_v0/base_L.obj` line 3 at r1](https://reviewable.io/reviews/RussTedrake/manipulation/262#gh-1372465400:-Nhg1Sd68Mj6568F1SV5:b-x8h36p) ([raw file](https://github.com/RussTedrake/manipulation/blob/a0db71001c33901a43ec10ddabd322750f4fdae0/manipulation/models/pr2_collision_convex/base_v0/base_L.obj#L3)):*
<details><summary><i>Previously, cohnt (Thomas Cohn) wrote…</i></summary><blockquote>

Switched back to just using the visual geometry as collision geometry, as in the original drake model.
</blockquote></details>

Thanks.  I only wonder if you want to make them as convex explicitly.  
___
*[`manipulation/models/pr2_collision_fixed.urdf` line 159 at r2](https://reviewable.io/reviews/RussTedrake/manipulation/262#-Nhg0tzr0MKsRPpuGS1C:-Nhg0tzs1aJJs7Tkx-3d:b-hv0u1u) ([raw file](https://github.com/RussTedrake/manipulation/blob/150d4c563c208c0af802319483882c1436dc02b5/manipulation/models/pr2_collision_fixed.urdf#L159)):*
> ```XML
>       <origin rpy="0 0 0" xyz="0 0 0"/>
>       <geometry>
>         <mesh filename="package://drake/examples/pr2/models/pr2_description/meshes/base_v0/base_L.obj"/>
> ```

does it make a difference if you mark these as convex? (not because they are convex, but because Drake will then use the convex hull)?

**Pull request merged**